internal/uploader/events_uploader.go

```diff
@@ -115,8 +116,14 @@ func (e *eventsUploader) sendRequest(req *http.Request) error {
 	// Successful http response code from the ingress server is in the 200s
 	if res.StatusCode >= 200 && res.StatusCode <= 299 {
 		e.log.Debugf("Successful response received from %s. Red Hat Insights Request ID: %+v", req.URL, res.Header.Get("X-Rh-Insights-Request-Id"))
+		return nil
 	}
-	return nil
+	defer res.Body.Close()
+	body, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		e.log.Debugf("error reading response body for request to %s", req.URL)
```

Comment:
Do you think we should also log the error here?

Reply:
I guess we can, since we're here :smile: 
Not really sure in what case this would error out though :sweat_smile: